.graphqlrc.yml

```diff
@@ -1 +1,12 @@
-schema: "https://vercel.saleor.cloud/graphql/"
\ No newline at end of file
+schema: https://vercel.saleor.cloud/graphql/
+documents: graphql/**/*.graphql
+extensions:
+  codegen:
+    overwrite: true
+    generates:
+      generated/graphql.ts:
+        plugins:
+          - typescript
+          - typescript-operations
+          - urql-introspection
+          - typescript-urql
```

Comment:
don't we need still `introspection` plugin to be run?

Reply:
@zaiste we have never used it; let's keep it simple and define only things that are needed